canonical-websites/www.ubuntu.com
# Add the user report page to desktop

## Done
Add the user report page to the desktop

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/statistics#desktop-specs](http://0.0.0.0:8001/statistics#desktop-specs)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check it matches [the design](https://camo.githubusercontent.com/c3c21357c308c5f12bdd0154416dd6919effaf86/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3561393533316162346235383036626332626338386433302f62326639643864392d613765352d343466332d623138332d333261303339303263383337)

## Screenshots
![screenshot_2018-10-18 user statistics ubuntu](https://user-images.githubusercontent.com/1413534/47147247-bca32980-d2c5-11e8-9ce3-7e15aa505fc2.png)
